implementation/GUI/gui.py

```diff
@@ -119,8 +138,8 @@ def not_ready_to_measure(self):
                 or len(self.scene.left_eye_points) != 2
                 or len(self.scene.right_eye_points) != 2
                 or len(self.scene.reference_points) != 4
-                or int(self.le_referenceMM.text()) < 1
```

Comment:
now our app accepts negative value for the length of a reference

Reply:
Sorry, I have removed the check for a positive value, but added a check for any value given. Now both conditions are checked